manifests/r/RazerInc/RazerInstaller/Synapse4/2.1.0.766/RazerInc.RazerInstaller.Synapse4.installer.yaml

```
@@ -0,0 +1,12 @@
+# Automatically updated by the winget bot at 2025/Feb/27
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: RazerInc.RazerInstaller.Synapse4
+PackageVersion: 2.1.0.766
+InstallerType: exe
+Installers:
+- Architecture: x86
+  InstallerUrl: https://rzr.to/synapse-4-pc-download
+  InstallerSha256: 95534154A32DE1E4B356F6DC37891227BCEED7D2125366575CC7913B5A21BE3C
```

```suggestion
  InstallerSha256: 4ED1C70EEA7477E018F15B750B0E10B1A3E055FEA44A5A676FDE2DDBE806104A
```

(Automated response - build 1028.)